templates/broadband-signup/inputs/payment-card.js

```diff
@@ -1,45 +1,20 @@
 import { templates } from '/src/main.js';
 
-function hostedForm(otp) {
-    // Styling and configuring the form
-    // https://docs.automationcloud.net/docs/vaulting-payment-card#section-styling-and-configuring-the-form
+// https://protocol.automationcloud.net/Generic#PaymentCard
+export default function hostedForm(otp) {
+    // Styling and configuring the form. See the README.
     const formConfiguration = {
-        // css (optional). An absolute URL
-        //
-        // https://docs.automationcloud.net/docs/vaulting-payment-card#section-css-optional-
-        css: 'https://ubio-application-bundle-dummy-server.glitch.me/style.css',
-
-        // fields (optional)
-        // https://docs.automationcloud.net/docs/vaulting-payment-card#section-fields-optional-
-        // https://docs.automationcloud.net/docs/vaulting-payment-card#section-interacting-with-the-form
-        //
-        // The form has 4 fields: name, pan (card number), expiry date, cvv.
-        // - change the order of fields
-        // - change the labels for fields
-        // - specify expiry date fields to be input (expiry) or select (expiry-select)
-        fields: 'pan,name,expiry-select,cvv',
-
-        // brands (optional).
-        //
-        // restricts aaceptable card brands list
-        brands: 'visa,mastercard,amex',
-
-        // validateOnInput (optional).
-        //
-        // 'on' to ebables form validation on input: whenever each field is updated, you receive the validation result message and an .invalid class gets added or removed accordingly
-        // 'off' disables form validation on user input, form will still be validated on submit
-        // Any other values will be ignored and the value will be set to 'off'.
-        validateOnInput: 'on'
+        css: 'https://ubio-application-bundle-test-server.glitch.me/style.css',
+        fields: ['pan', 'name', 'expiry-select', 'cvv'],
+        brands: ['visa', 'mastercard', 'amex'],
+        validateOnInput: true
```

Comment:
my only concern is that we introduce a mismatch between this and official docs. could it be confusing? I like this approach more, just unsure should we use same names for params but different values (like 'on' became true) or it's nicer to slightly modify names to clarify it's not 1:1 (e.g `fields` become `fieldsInOrder` or something like that) ??

Reply:
It could be, yes. However, the official docs are for a query string, so it's already a little weird to split this into css, fields, and brands options. It should probably be one way or the other. Right now on master it's both.